dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4115,6 +4115,65 @@ paths:
       - bearer: []
       tags:
       - workflows
+  /workflows/{workflowId}/refresh:
+    get:
+      description: Refresh one particular workflow.
+      operationId: refresh
+      parameters:
+      - in: path
+        name: workflowId
+        required: true
+        schema:
+          format: int64
+          type: integer
+      requestBody:
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/User'
+      responses:
+        default:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Workflow'
+          description: default response
+      security:
+      - bearer: []
+      tags:
+      - workflows
+  /workflows/{workflowId}/refresh/{version}:
+    get:
+      description: Refresh one particular workflow version.
+      operationId: refreshVersion
+      parameters:
+      - in: path
+        name: workflowId
+        required: true
+        schema:
+          format: int64
+          type: integer
+      - in: path
+        name: version
+        required: true
+        schema:
+          type: string
+      requestBody:
```

Comment:
user not hidden for both endpoints

Reply:
How do you hide with openapi?

Reply:
`@Parameter(hidden = true, name = "user", in = ParameterIn.HEADER)`